adapter/service/outlvlswitch/event.go

```diff
@@ -0,0 +1,30 @@
+package outlvlswitch
+
+import (
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/event"
+)
+
+type LevelEvent struct {
```

Comment:
Instead of creating a separate type here, I'd add a `payload` field to `ServiceEvent` and I'd stick to the payload types returned from Controllers/Reporters, so in this case it would be simply an `int64`. I think this convention will be simple to follow.